@xen-orchestra/proxy/src/app/mixins/reverseProxy.mjs

```diff
@@ -0,0 +1,117 @@
+import { urlToHttpOptions } from 'url' // node  14.18.0, 15.7.0 is it ok ?
```

Comment:
Update Node requirement in `package.json`.

Reply:
I wuold like to have `    "node": ">= 14.18 < 15.0 || >= 15.7"` but I keep having `error  Parsing error: @babel/helper-compilation-targets: '14.18 <15.0 || >=15.7' is not a valid value for 'targets.node'` error on lint

seems like [[const versionRegExp = /^(\d+|\d+.\d+)$/; ](https://github.com/babel/babel/blob/cb3ebde8ceed47f4b6783c16137cc9ddc78998b7/packages/babel-helper-compilation-targets/src/utils.ts#L8) ](https://github.com/babel/babel/blob/cb3ebde8ceed47f4b6783c16137cc9ddc78998b7/packages/babel-helper-compilation-targets/src/utils.ts#L8) is looking for one version

in this case I would like to accept 14.18 and decline 15.1

Reply:
At least require Node 14.18